MC/bin/o2dpg_sim_workflow_anchored.py

```diff
@@ -232,19 +232,22 @@ def retrieve_MinBias_CTPScaler_Rate(ctpscaler, finaltime, trig_eff, NBunches, Co
     if ColSystem == "PbPb":
       ctpclass = 25  # <--- we take scalers for ZDC
       ctptype = 7
-    print("Fetching rate with class " + str(ctpclass) + " type " + str(ctptype))
+    print("Fetching rate with time " + str(finaltime) + " class " + str(ctpclass) + " type " + str(ctptype))
     rate = ctpscaler.getRateGivenT(finaltime, ctpclass, ctptype)
     #if ColSystem == "PbPb":
     #  rate.first = rate.first / 28.
     #  rate.second = rate.second / 28.
 
     print("Global rate " + str(rate.first) + " local rate " + str(rate.second))
+    scaler = None
+    if rate.second >= 0:
+      scaler = rate.second
```

Comment:
I guess this is a matter of taste. But what `ctpscaler` returns are the scaler values or some kind of counts from its trigger inputs. So how about `ctp_rate`?

Reply:
But `finalRate` also uses `rate.second` which you refer to as `local_rate`

Reply:
Yes, you are right, `finalRate` is *computed* from `rate.second`.
When I read this (https://github.com/AliceO2Group/AliceO2/blob/dev/DataFormats/Detectors/CTP/include/DataFormatsCTP/Scalers.h#L126-L127), what you return is the bare/raw local rate in some sense. I think, it could be called something like `local_rate_raw` or `ctp_local_rate_raw` or something.